### PR TITLE
Add "prefix" operator

### PR DESCRIPTION
**Pull request by achoum**
Please review from PR by comparing it to branch `gbm_new_assign`

### CODE REVIEW

**Review by diegomarvid — APPROVED** [review]
LGTM 😄

**Review by ianspektor — APPROVED** [review]

File: temporian/implementation/numpy/operators/prefix.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        prefix = self._operator.prefix()
+        dst_event = NumpyEvent(data={}, sampling=event.sampling)
+
+        # For each index value
+        for index, src_mts in event.data.items():
+            dst_mts = []
+            dst_event.data[index] = dst_mts
+
+            # For each feature
+            for src_ts in src_mts:
```
[the remark this answers is not in the document]
Comment (achoum) [reply temporian/implementation/numpy/operators/prefix.py:32]: Sounds great :).
Thanks for the code snippet. I'll make sure to use this in the future.

File: temporian/core/operators/prefix.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    event: Event,
+) -> Event:
+    """Add a prefix to the feature names.
+
+    Example:
+
+        Inputs:
+            prefix: "hello_"
+            event:
+                feature_1: ...
```
[the remark this answers is not in the document]
Comment (achoum) [reply temporian/core/operators/prefix.py:94]: Good point.
*Done

**Comment by achoum** [comment]
Thanks

**Pull request merged**